docs/reference/developer/getting-started.rst

```diff
@@ -70,7 +70,7 @@ Alternatively, to build a release version with all optimizations enabled:
 
   cmake \
     -GNinja \
-    -DPYTHON_EXECUTABLE=$(command -v python3) \
+    -DPython_EXECUTABLE=$(command -v python3) \
```

Comment:
Should also change in https://github.com/scipp/scipp/blob/main/tools/build_cpp.py#L50